src/theme/default/colors.ts

```diff
@@ -42,10 +42,10 @@ const rawColors: RawColorsT = {
 
 const appColors: AppColorsT = {
   borderColor: rawColors.gray.bombay,
-  controlFocused: rawColors.gray.limedSpruce,
+  controlFocused: rawColors.gray.darkMud,
   error: rawColors.red.redOrange,
   success: rawColors.green.greenHaze,
-  text: rawColors.gray.limedSpruce,
+  text: rawColors.gray.darkMud,
```

Comment:
Was the previous value wrong?

Reply:
All the latest mocks have this color for text (as well as `HeadPanel` etc...)

Reply:
Resolving after discussion with @oyabun concluding in using 335414A